Makefile

```diff
@@ -0,0 +1,43 @@
+# Runs the utility to pull in all header files from `google/mediapipe`
+headers:
+	cd packages/build_cmd && dart bin/main.dart headers
+
+# Runs `ffigen` for all packages
+generate: generate_core generate_text
+
+# Runs `ffigen` for all packages, compiles the faked C artifacts, and runs all tests
+test: generate_text test_text generate_core test_core
+
+# Runs `ffigen` for all packages and all tests for all packages
+test_only: test_core test_text
```

Comment:
```suggestion
test_all: test_core test_text
```
actually?

Reply:
Could be this, for sure. My thinking was that `make test` should do *everything*, which in our case includes bringing in headers, running `ffigen`, and compiling the C fakes. But, if someone knows those prereqs are all in good shape, then a quicker command to *only* run the tests would also be nice.

Open to other ideas on this line of thinking.